wikiclue/src/routes/home/+page.svelte

```diff
@@ -8,12 +8,33 @@
     import Overlay from "../../components/overlay.svelte";
     import { goto } from '$app/navigation';
 	import { writable } from "svelte/store";
+    import { browser } from '$app/environment';
+    import { rush } from "../../store/gameplay";
 
     export let levelsOpen = false;
     export let dailyOpen = false;
     export let rushOpen = false;
-
     const isOverlayOpen = writable(false);
+
+    function playRush() {
+        if (browser) {
```

Comment:
This checks if the app is running in the browser, why is it needed? I would think it would be given if playRush is called